crates/loam-cli/tests/it/build_clients/contracts.rs

```diff
@@ -165,11 +163,32 @@ soroban_token_contract.client = false
             .output()
             .expect("Failed to execute command");
 
-        println!("stderr: {}", String::from_utf8_lossy(&output3.stderr));
         // ensure contract hash change check works, should update in dev mode
         assert!(output3.status.success());
-        assert!(String::from_utf8_lossy(&output3.stderr)
-            .contains("🔄 Updating contract \"hello_world\""));
+        let message = String::from_utf8_lossy(&output3.stderr);
+        assert!(message.contains("🔄 Updating contract \"hello_world\""));
+        let Some(contract_id) = extract_contract_id(&message) else {
+            panic!("Could not find contract ID in stderr");
+        };
+        env.set_environments_toml(format!(
+            r#"
+production.accounts = [
+    {{ name = "alice" }},
+]
+
+[production.network]
+rpc-url = "http://localhost:8000/rpc"
+network-passphrase = "Standalone Network ; February 2017"
+
+[production.contracts]
+hello_world.client = true
+hello_world.id = "{}"
```

Comment:
Do we still need the `client = true` if another setting is present?

Reply:
I just pushed an update to default it to true if others are present